tests/test_mlkit.py

```diff
@@ -420,6 +446,26 @@ def test_model_format_setters(self):
             }
         }
 
+    def test_model_as_dict_for_upload(self):
+        model_source = mlkit.TFLiteGCSModelSource(gcs_tflite_uri=GCS_TFLITE_URI)
+        model_format = mlkit.TFLiteFormat(model_source=model_source)
+        model = mlkit.Model(display_name=DISPLAY_NAME_1, model_format=model_format)
+        assert model.as_dict(for_upload=True) == {
+            'displayName': DISPLAY_NAME_1,
+            'tfliteModel': {
+                'gcsTfliteUri': GCS_TFLITE_SIGNED_URI
+            }
+        }
+
+    @pytest.mark.parametrize('helper_func', [
```

Comment:
This test assumes TF is not installed in the runtime.  Can we make it more reliable?

```
try:
  import tensorflow
  mlkit._TF_ENABLED = False
except:
  pass
```

Reply:
We can just set _TF_ENABLED to false regardless of importing tensorflow or not. Then the test will always raise the exception.